templates/profile/prometheus/rules.yml.erb

```diff
@@ -180,6 +180,13 @@ groups:
     for: 30m
     labels:
       severity: ticket
+  - alert: HTMountReadOnly
+    annotations:
+      summary: '{{$labels.hostname}}:{{$labels.mountpoint}} is READ ONLY.'
+    expr: 'node_filesystem_readonly{role=~"nebula::role::webhost::htvm.*",mountpoint="/htapps"} > 0'
+    for: 30m
```

Comment:
I would recommend alerting much sooner than this, even with no `for`, or if you definitely need one `for: 1m`. This is not an expected condition at all, and if a web server goes in the pool in this state it will result in users getting 500 errors.

Reply:
I pulled those params from a different alert.   a quicker alert makes sense.   I also added it as a `page` rather than just a `ticket`, due to feeling like this is more of an urgent matter, rather than something that can wait until monday. Do you agree with that?

Reply:
Agreed, this is a pants-on-fire, fix ASAP or support tickets start flooding in about errors.